README.md

```diff
@@ -25,14 +25,21 @@ By convention, the file should have the extension `.sspak` for non-executable ve
 
 You can run the installation script one of three ways.
 
-### Composer
+### Composer (recommended)
 
 You can install this package globally with Composer (ensure your composer bin is in your system path):
 
     $> composer global require silverstripe/sspak:dev-master
 
+You can also require it directly in your project
+
+    $> composer require --dev silverstripe/sspak
+    $> vendor/bin/sspak <commands>
+
```

Comment:
Could we mention that user might see a message `mysqldump: unknown variable 'column-statistics=0'`